File: VB/CustomWizardExample/Wizard/Presenters/ChoosePersonsPage3.vb
Imports Microsoft.VisualBasic
Imports System
Imports CustomWizardExample.Wizard.Views
Imports DevExpress.Data.WizardFramework
Imports DevExpress.DataAccess.Wizard.Model
Imports DevExpress.XtraReports.Wizards3

Namespace CustomWizardExample.Wizard.Presenters
	Public Class ChoosePersonsPage3
		Inherits ChoosePersonsPage2(Of XtraReportModel)
		Public Sub New(ByVal view As Views.IChoosePersonsPageView)
			MyBase.New(view)
		End Sub

		Public Overrides Function GetNextPageType() As Type
			Return GetType(DevExpress.XtraReports.Wizards3.Presenters.SelectColumnsPage(Of XtraReportModel))
		End Function
		Public Overrides ReadOnly Property MoveNextEnabled() As Boolean
			Get
				Return True
			End Get
		End Property
		Public Overrides ReadOnly Property FinishEnabled() As Boolean
			Get
				Return False
			End Get
		End Property
	End Class
End Namespace

File: CS/CustomWizardExample/Wizard/Presenters/ChoosePersonsPage3.cs
using System;
using CustomWizardExample.Wizard.Views;
using DevExpress.Data.WizardFramework;
using DevExpress.DataAccess.Wizard.Model;
using DevExpress.XtraReports.Wizards3;

namespace CustomWizardExample.Wizard.Presenters {
    public class ChoosePersonsPage3 : ChoosePersonsPage2<XtraReportModel> {
        public ChoosePersonsPage3(Views.IChoosePersonsPageView view) : base(view) { }

        public override Type GetNextPageType() { return typeof(DevExpress.XtraReports.Wizards3.Presenters.SelectColumnsPage<XtraReportModel>); }
        public override bool MoveNextEnabled { get { return true; } }
        public override bool FinishEnabled { get { return false; } }
    }
}
